docs/en/latest/plugin-develop.md

```diff
@@ -493,4 +493,12 @@ function _M.control_api()
 end
 ```
 
-If you don't change the default control API configuration, the plugin will be expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`.
+If you don't change the default control API configuration, the plugin will expose `GET /v1/plugin/example-plugin/hello` which can only be accessed via `127.0.0.1`. Test with the following command:
+
+```shell
+curl -i -X GET "http://localhost:9090/v1/plugin/example-plugin/hello"
+```
+
+:::note Note
```

Comment:
Is it necessary to use `:::note`? If not, we can remove it.

Reply:
To me, this is just a regular tip. Better leave it for something really critical.